apps/ui/src/components/EditorVote.vue

```diff
@@ -0,0 +1,128 @@
+<script setup lang="ts">
```

Comment:
To me this component should be called `EditorVotingType` and there should be a second one `EditorChoices`